config.go

```diff
@@ -443,8 +469,10 @@ type Config struct {
 	//
 	// Optional: defaults to 200
 	DataIterationBatchSize uint64
```

Comment:
This is just a comment, no action is needed for now: I can kinda see that if we moved this variable into the struct with the rest of the DataIterationBatchSize config, we can refactor some of the code a bit better (including things like GetBatchSize from CursorConfig, to the not-yet created DataIterationBatchSizeConfig struct). 

For a later time, perhaps.